tools/pd-recover.md

```diff
@@ -0,0 +1,50 @@
+---
+ title: PD Recover User Guide
+ summary: Use PD Recover to recover a PD cluster which cannot start or provide services normally.
+ category: tools
+ ---
+
+ # PD Recover User Guide
+
+ As a disaster recovery tool of PD, PD Recover recovers the PD cluster which cannot start or provide services normally.
+
+ ## Source code compiling
+
+1. [Go](https://golang.org/) Version 1.9 or later
+2. In the PD root directory, use the `make` command to compile and generate `bin/pd-recover`
+
+ ## Usage
+
+ ### Flags description
+
+ ```
+   -alloc-id uint
+         Specify a number larger than the allocated ID of the original cluster
+
+   -cacert string
+         Specify the path to the trusted CA certificate file in PEM format
+
+   -cert string
+         Specify the path to the SSL certificate file in PEM format
+
+   -key string
+         Specify the path to the SSL certificate key file in PEM format, which is the private key of the certificate specified by `--cert`
+
+   -cluster-id uint
+         Specify the Cluster ID of the original cluster
+
+   -endpoints string
+         Specify the PD address (default："http://127.0.0.1:2379")
+ ```
+
+ ### Recovery flow
+
+ 1. Obtain the Cluster ID and the Alloc ID from the current cluster. 
+ 
+     Generally, you can obtain the Cluster ID from the PD, TiKV and TiDB log and the allocated Alloc ID from either the PD log or the `Metadata Information` in the PD monitoring panel. 
+     
+     Specifying `alloc-id` requires a number larger than the current largest Alloc ID. If you fail to obtain the Alloc ID, you can make an estimate of a larger number according to the number of Regions and Stores in the cluster. Normally, you can specify a number that is several orders of magnitude larger.
```

Comment:
Pls limit the use of the word "normally" in TiDB documentation. "Normally" -> **Generally**